src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/osgi/OSGiConfiguration.kt

```diff
@@ -0,0 +1,38 @@
+package com.cognifide.gradle.aem.common.instance.service.osgi
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+class OSGiConfiguration {
+
+    @JsonProperty
+    var pid: String = ""
+
+    @JsonProperty("properties")
+    private var configProperties: Map<String, ConfigurationProperty> = mutableMapOf()
+
+    @JsonProperty
+    var bundleLocation: String = ""
```

Comment:
if it could be null use `var bundleLocation: String? = null`
if not use `lateinit var bundleLocation: String`